src/lib.rs

```diff
@@ -286,6 +330,8 @@ impl Debug for Value {
             Value::Bool(b) => write!(fmt, "(bool {})", b),
             Value::String(s) => write!(fmt, "(string \"{}\")", s),
             Value::List(v) => write!(fmt, "(array {:?})", v),
+            Value::Set(v) => write!(fmt, "(set {:?})", v),
+            Value::Dict(v) => write!(fmt, "(dict {:?})", v),
```

Comment:
These prints are a bit annoying. It would be great to just write the content, maybe a few borrows?